tests/Functional/tests/standalone-email-addresses/standalone-email-addresses.html

```diff
@@ -79,8 +79,15 @@ <h1>
 <p>Appendix A.4. Messages with Trace Fields</p>
 <blockquote>
     <hr />
-    <p>Received: from x.y.test by example.net via TCP with ESMTP id ABC12345 for &lt;<a href="mailto:mary@example4.net">mary@example4.net</a>&gt;; 21 Nov 1997 10:05:43 -0600
-Received: from node.example by x.y.test; 21 Nov 1997 10:01:22 -0600
+    <dl>
```

Comment:
Honestly, I'm blown away by this test file. It was added to test the automatic recognition of e-mailaddresses. However, testing raw emails headers as reStructuredText seems very weird (resulting in strange things like this definition list).